eq-author-api/scripts/import-json-to-dynamo.js

```diff
@@ -9,7 +9,7 @@ if (!jsonPath) {
 }
 
 const importQuestionnaire = async contents => {
-  return createQuestionnaire({ ...contents, id: contents.id + "_copy" });
+  return createQuestionnaire({ ...contents, id: contents.id });
```

Comment:
So this was the only bit of the problem yesterday? It can't have `_`? Or was that it wasn't a hex value?

Reply:
It was the `_` the regex creating the url was ignoring it

Reply:
That plus the updated at stuff was also causing another issue